core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -259,6 +154,7 @@ public void resetCallbacksForWorkflow(String workflowId) {
                         });
     }
 
+    @PreAuthorize("hasPermission(#request, 'OPERATOR')")
```

Comment:
Can we use SpEL to define the permission instead of 'OPERATOR' string?

Reply:
Good catch. Updated to use SpEL.